pkg/controller/volume/persistentvolume/index.go

```diff
@@ -170,7 +167,7 @@ func (pvIndex *persistentVolumeOrderedIndex) findByClaim(claim *v1.PersistentVol
 
 // findBestMatchForClaim is a convenience method that finds a volume by the claim's AccessModes and requests for Storage
 func (pvIndex *persistentVolumeOrderedIndex) findBestMatchForClaim(claim *v1.PersistentVolumeClaim) (*v1.PersistentVolume, error) {
-	return pvIndex.findByClaim(claim, matchStorageCapacity)
+	return pvIndex.findByClaim(claim)
 }
 
 // matchStorageCapacity is a matchPredicate used to sort and find volumes
```

Comment:
Ack. Maybe move `matchStorageCapacity(...)` to `persistentvolume/index_test.go` then?